tests/installation/start_install.pm

```diff
@@ -82,7 +83,21 @@ sub run {
         assert_screen("inst-packageinstallationstarted", $started_timeout);
     }
     else {
-        sleep 2;    # textmode is sometimes pressing alt-i too early
+        if (check_var('DESKTOP', 'kde')) {
+            send_key_until_needlematch 'packages-section-selected', 'tab';
+            send_key 'ret';
+            send_key_until_needlematch 'gnome-desk-env', 'down';
+            send_key ' ';    #deselect gnome
+            send_key_until_needlematch 'kde-desk-env', 'down';
+            send_key ' ';    #select kde
+            send_key 'alt-o';
+            if (check_screen('confirmlicense', 5)) {
```

Comment:
Please use a multi-tag `assert_screen` with `match_has_tag` instead of `check_screen` with non-zero timeout to prevent introducing any timing dependant behaviour, to save test execution time as well as state more explicitly from the testers point of view what are the expected alternatives. For example:

```perl
assert_screen([qw(yast2_console-finished yast2_missing_package)]);
if (match_has_tag('yast2_missing_package')) {
    send_key 'alt-o';  # confirm package installation
    assert_screen 'yast2_console-finished';
}
```

Reply:
Code updated.